modules/op-editing-pipelines.adoc

```diff
@@ -12,7 +12,7 @@ You can edit the Pipelines in your cluster using the *Developer* perspective of
 
 . In the *Pipelines* view of the *Developer* perspective, select the Pipeline you want to edit to see the details of the Pipeline.
 In the *Pipeline Details* page, click *Actions* and select *Edit Pipeline*.
-. In the *Pipeline Builder* page:
+. In the *Pipeline builder* page:
```

Comment:
@masauso-lungu **Suggestion:** Use a complete sentence for the lead-in sentence to reduce ambiguity and support translation.

Refer to https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences